pkg/docker/docker.css

```diff
@@ -1,3 +1,18 @@
+@import "/plot.css";
+@import "/term.css";
```

Comment:
Out of interest, why were these two lines added here? Not necessarily a problem, but seem to be unrelated to this pull request.